block/retriever.go

```diff
@@ -50,19 +50,22 @@ func (m *Manager) syncUntilTarget(ctx context.Context, syncTarget uint64) error
 			return err
 		}
 
-		if settlementBatch.StartHeight != currentHeight+1 {
+		//code commented out because it may be the case the start height from the da batch is not exactly the next block from the last received
```

Comment:
you can simply delete the entire lines. no need for leaving it with comments.

Reply:
done